test/WebApiShimFw46.FunctionalTests/project.json

```diff
@@ -17,9 +17,10 @@
     "Microsoft.AspNetCore.StaticFiles": "1.0.0-rc2-final",
     "Microsoft.AspNet.WebApi.Client": "5.2.2",
     "Microsoft.Extensions.Configuration.Json": "1.0.0-rc2-final",
-    "dotnet-test-xunit": "1.0.0-rc2-187791-49",
+    "dotnet-test-xunit": "1.0.0-rc3-*",
     "xunit": "2.1.0",
-    "Microsoft.VisualStudio.Web.CodeGeneration.Utils": "1.0.0-preview1-final"
+    "Microsoft.VisualStudio.Web.CodeGeneration.Utils": "1.0.0-preview1-final",
+    "Microsoft.NETCore.Platforms": "1.0.1-rc2-24027"
```

Comment:
These netcore platform chagnes are made to check if the tests work for net451 on build machine.